src/main/java/com/iridium/iridiumskyblock/listeners/PlayerChatListener.java

```diff
@@ -16,30 +17,30 @@
 
 public class PlayerChatListener implements Listener {
 
-    @EventHandler
+    @EventHandler(ignoreCancelled = true)
```

Comment:
Ignoring cancelled event calls results in players being able to bypass mutes etc

Reply:
```suggestion
    @EventHandler
```

Reply:
I dont see how this would cause any issues? it just wont run our code if the event is cancelled, it basically just means that the players wont be able to use /is chat if they are muted

Reply:
Oh, I see, my bad